sandbox/apparmor/apparmor.go

```diff
@@ -460,6 +460,7 @@ func AppArmorParser() (cmd *exec.Cmd, internal bool, err error) {
 			args := []string{
 				"--config-file", filepath.Join(prefix, "/apparmor/parser.conf"),
 				"--base", filepath.Join(prefix, "/apparmor.d"),
+				"--include", "/etc/apparmor.d/tunables/home.d",
```

Comment:
This is needed to correctly support relative #include statements from files inside `home.d` here